src/v/redpanda/application.cc

```diff
@@ -559,6 +559,24 @@ void application::check_environment() {
     storage::directories::initialize(
       config::node().data_directory().as_sstring())
       .get();
+
+    if (config::shard_local_cfg().storage_strict_data_init()) {
+        // Look for the special file that indicates a user intends
+        // for the found data directory to be the one we use.
+        auto strict_data_dir_file
+          = config::node().strict_data_dir_file_path().string();
+        auto file_exists = ss::file_exists(strict_data_dir_file).get0();
```

Comment:
nit: use `get()` instead of `get0()` which will be deprecated eventually

Reply:
Switching to `get`. Nice catch!